jep/1/README.adoc

```diff
@@ -112,22 +115,14 @@ There are three kinds of JEP:
 
```

Comment:
If BDFL has no responsibilities, do we even need to keep the role?   

No offence intended to Kohsuke, but why have a role defined for this process that has no active role in this process?

Reply:
I have contacted KK several days ago about the future of the BDFL role. Just FYI, pending discussion